modules/p2/ding_serendipity_taxonomy_term/ding_serendipity_taxonomy_term.module

```diff
@@ -66,34 +70,44 @@ function ding_serendipity_taxonomy_term_recommended_objects_serendipity_add($con
  * Fetch recent content from Brønd relating to term in context.
  */
 function ding_serendipity_taxonomy_term_recent_serendipity_add($context) {
-  $term_arr = ding_serendipity_taxonomy_get_term($context);
+  $term_array = ding_serendipity_taxonomy_get_term($context);
 
   // Validate context.
-  if ($term_arr === FALSE || !isset($context['recent'])) {
+  if ($term_array === FALSE || !isset($context['recent'])) {
     return array();
   }
 
-  $facets = array();
-  $term = $term_arr['term'];
-
+  $query = _ding_serendipity_taxonomy_term_query_from_term($term_array);
   // Select content from this year.
   $limit = (is_numeric($context['max'])) ? $context['max'] : 16;
+  $query->setCount($limit);
 
-  $term_arr['query'] = ding_list_convert_cql($term_arr['query']);
-
-  // Apply term query.
-  $facets[] = $term_arr['query'];
+  // Order by date.
+  $query->addSort(TingSearchCommonFields::ACQUISITION_DATE, TingSearchSort::DIRECTION_DESCENDING);
```

Comment:
TingSearchCommonFields::ACQUISITION_DATE can be remove here as it has no effect in the data-well.

Reply:
What do you suggest we do here then - remove all references to `TingSearchCommonFields::ACQUISITION_DATE`?

Reply:
Lets leave it for now. It don't break anything, it just don't have any effect in DDB/FBS context.